Makefile

```
@@ -1,3 +1,6 @@
+# The number of CPUs available.
+ncpus = $$(nproc)
```

`nproc` doesn't work under macOS. For a basic cross-platform implementation:

```suggestion
ncpus = $$(which nproc && nproc || sysctl -n hw.logicalcpu)
```